query/Cargo.toml

```diff
@@ -19,6 +19,7 @@ fuse-query-metrics = {path = "metrics"}
 
 # github dependencies, in alphabetical order
 arrow = { git = "https://github.com/apache/arrow", rev="46161d2" }
+parquet = { git = "https://github.com/apache/arrow", rev="46161d2" }
```

Comment:
what does this mean? import arrow with an alias name?

Reply:
No, because `parquet` is the member one of  `https://github.com/apache/arrow/blob/master/rust/Cargo.toml`.

It's the same level as crate `arrow` and `datafusion ` etc...

Reply:
I see Thanks! :DDD